x-pack/plugins/viz_editor/server/query/to_table.test.ts

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Query } from '../../common';
+import { toTable } from './to_table';
+
+describe('viz-editor/query/to_table', () => {
```

Comment:
I don't see any tests for tabularizing a nested field, are we going to handle that?

Reply:
If I understand you correctly, I think this is an example of what you mean by a "nested field":

```js
{
  select: [
    { operation: 'col', argument: 'products.category.keyword' },
    { operation: 'count' },
  ],
}
```

There, `prodcuts.category` is nested (products is an array in each document, and category is a property on each object in that array). But the result is shaped like several in our test suite:

```json
  "aggregations" : {
    "groupby" : {
      "after_key" : {
        "prod" : "Women's Shoes"
      },
      "buckets" : [
        {
          "key" : {
            "prod" : "Men's Accessories"
          },
          "doc_count" : 572,
          "count" : {
            "value" : 572
          }
        },
```

So, I *think* we're good. I'm sure we'll have lots of edge cases over time that aren't in here yet, though!